### PR TITLE
Card

### PR DESCRIPTION
**Pull request by JuvitaSaini09**
1. added card component
2.  added documentation of card component
image

### CODE REVIEW

File: components/card/card.html [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+                    style=" height: 200px;  border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </div>
+
+
+            <h3 class="h3">Text only card</h3><br>
```
Comment (gauravsinhaweb) [inline components/card/card.html:150]: ig there is no use of <br/> here

File: components/card/card.html [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+                </iframe>
+            </div>
+
+            <!--Card 2-->
+           
+            <h2 class="grey mb2">Cards with badge and dismiss</h2>
+            <div class=" card card2">
+                <div class="card-badge">
+                    <span class="sale-badge">on Sale</span>
+                    <button class="dismiss-badge">X</button>
```
Comment (gauravsinhaweb) [inline components/card/card.html]: use cross icon here

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]

File: components/card/card.html (hunk 1 above)
Comment (JuvitaSaini09) [reply components/card/card.html:150]: its sub-heading

File: components/card/card.html (hunk 2 above)
Comment (JuvitaSaini09) [reply components/card/card.html]: i have added the icons,thankyou for the review

File: components/card/card.css [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+      padding-top:3px;
+      font-weight:500;
+    position: absolute;
+    top: -3px;
+    left: -5px;
+    height: 1.7rem;
+    width: 4.7rem;
+    text-align: center;
+    border: 1px solid var(--orange);
+    background-color: (--gold);
```
Comment (amantulsyan35) [inline components/card/card.css]: there should be a var here

**Review by gauravsinhaweb — APPROVED** [review]

File: components/card/card.css [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+   --red:red;
+   --white:white;
+   --pink:pink;
+   --grey:grey;
+
+}
+
+.card {
+    position:relative;
+    margin-top: 20px;
```
Comment (amantulsyan35) [inline components/card/card.css]: instead of px rem would be better

File: components/card/card.css [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+  .dismiss-badge:hover{
+    background-color:var(--pink);
+    
+  }
+  .text-overlay{
+    font-weight:bold;
+    font-size:26px;
+    padding-top:6rem;
+    text-align:center;
+    background-color:(--grey);
```
Comment (amantulsyan35) [inline components/card/card.css]: var here also, it will make the CSS code more consistent

File: components/card/card.css (hunk 3 above)
Comment (JuvitaSaini09) [reply components/card/card.css]: fixed

File: components/card/card.css (hunk 5 above)
Comment (JuvitaSaini09) [reply components/card/card.css]: fixed

File: components/card/card.css (hunk 4 above)
Comment (JuvitaSaini09) [reply components/card/card.css]: i have made the changes,please approove my PR

**Review by mehulsatardekar — APPROVED** [review]
Apart from the changes the people have suggested above, the code looks good to merge from my side

File: components/card/card.css [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+    margin-top: 1.24rem;
+    position: relative;
+    height: 24rem;
+    width: 12rem;
+    border-radius: 10px;
+    box-shadow: rgba(0, 0, 0, 0.25) 0px 14px 28px,
+      rgba(0, 0, 0, 0.22) 0px 10px 10px;
+  }
+  .sale-badge {
+      padding-top:3px;
```
Comment (AshutoshM29) [inline components/card/card.css:24]: I would suggest you should try to use rem here too, just to keep the code uniform

File: components/card/card.html [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+                    style=" height: 200px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </div>
+
+
+
+            <!-- card3-->
+            
+            <h2 class="grey mb2">Vertical Card with text-overlay</h2>
```
Comment (AshutoshM29) [inline components/card/card.html:121]: You can change the name of class
gray sounds vague

**Review by AshutoshM29 — CHANGES_REQUESTED** [review]
Rest looks good
You can merge it

**Pull request merged**